MediaBrowser.Api/EnvironmentService.cs

```diff
@@ -52,6 +52,7 @@ public class ValidatePath
         public bool? IsFile { get; set; }
     }
 
+    [Obsolete]
```

Comment:
why? there is a feature where we expose alternative path for clients who can handle that (like Kodi), is it another feature?

Reply:
I'm fairly certain it's a different feature.